starknet_py/hash/utils.py

```diff
@@ -0,0 +1,40 @@
+import functools
+from typing import Optional, Sequence
+
+from crypto_cpp_py.cpp_bindings import ECSignature, cpp_hash
+from eth_utils.crypto import keccak
+from starkware.cairo.lang.vm.crypto import pedersen_hash as default_hash
+from starkware.crypto.signature.signature import sign
+
+from starknet_py.common import int_from_bytes
+from starknet_py.constants import MASK_250
+from starknet_py.utils.crypto.facade import use_cpp_variant
+
+
+def _starknet_keccak(data: bytes) -> int:
+    """
+    A variant of eth-keccak that computes a value that fits in a StarkNet field element.
+    """
+    return int_from_bytes(keccak(data)) & MASK_250
+
+
+def pedersen_hash(left: int, right: int) -> int:
+    if use_cpp_variant():
+        return cpp_hash(left, right)
+    return default_hash(left, right)
```

Comment:
Let's add some kind of test for it.

Reply:
Brother 💀 I was just moving stuff around in this pr.